en/private-endpoints.html

```diff
@@ -0,0 +1,125 @@
+      See <a href="https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html">endopint service configuration</a>
+      for details on valid ARNs, and more fine-grained access control.
+    </p>
+<pre>
+&lt;deployment version='1.0'&gt;
+  &lt;prod&gt;
+    &lt;region&gt;region-1&lt;/region&gt;
+    &lt;region&gt;region-2&lt;/region&gt;
+  &lt;/prod&gt;
+  &lt;endpoints&gt;
+    &lt;endpoint type='private' container-id='my-container'&gt;
+      &lt;allow with='aws-private-link' arn='arn:aws:iam::123123123123:root' /&gt;
+    &lt;/endpoint&gt;
+  &lt;/endpoints&gt;
+&lt;/deployment&gt;
+</pre>
+    <p>
+      Build and submit the application package, and wait for it to deploy to the indicated regions. 
+    </p>
+  </li>
+  <li>
+    <p><b>Find the service ID of your endpoint services</b></p>
+    <p>
+      Navigate to the endpoints tab for your application, in the Console, and find the 
+      service ID for the deployment to wish to connect to. While there, verify that access to
+      connect to the endpoint was granted to the correct ARNs.
+    </p>
+  </li>
+  <li>
+    <p><b>Create the VPC interface endpoint</b></p>
+    <p>
+      <a href="https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc-endpoint.html">Create a VPC endpoint</a>
+      in your VPC. This is the entry point for clients that need to connect to the endpoint service, which lets them
+      connect to the Vespa cluster through the private network of AWS. For this example, assume your VPC has id
+      <code>vpc-123</code> and resides in the AWS region <code>us-east-1</code>, and that the service ID of your
+      endpoint service, found in the Console, is <code>com.amazonaws.vpce.us-east-1.vpce-svc-321</code>:
+    </p>
+<pre>
+$ aws ec2 create-vpc-endpoint --region us-east-1 --vpc-id vpc-123 --service-name com.amazonaws.vpce.us-east-1.vpce-svc-321 --vpc-endpoint-type Interface --private-dns-enabled | jq .
+</pre>
+    <p>
+      Note the value of the <code>VpcEndpointId</code> field, for verification in the below item.
+      This is also where you specify optional security group and subnet IDs; these are omitted here for brevity.
+      If creating the VPC endpoint through the AWS console instead, be sure to check "Enable DNS names"!
+    </p>
+  </li>
+  <li>
+    <p><b>Verify the VPC endpoint is connected to the Vespa cluster</b></p>
+    <p>
+      Navigate back to the endpoints tab in the Console, and refresh the page. You should now see a new entry representing
+      the connection between your newly created interface endpoint, and the endpoint service on your container cluster;
+      verify the ID matches the value of the <code>VpcEndpointId</code> field above.
+      The connection is ready when the state is <code>available</code>.
+    </p>
+  </li>
+  <li>
+    <p><b>Verify your Vespa cluster is reachable from within your VPC</b></p>
```

Comment:
&lt;b&gt; is not recommended in HTML5:
```suggestion
    <p><strong>Verify your Vespa cluster is reachable from within your VPC</strong></p>
```